examples/datacalc.py

```diff
@@ -137,12 +142,8 @@ def setDB(self, name: str):
         """
         dbBox = self.viewerFrame.dbBoxes[name]
         self.dbNames[name] = dbBox.currentText()
-        self.broadcast(
-            "log", 
-            f"Database {name} is set as {self.dbNames[name]}."
-            if self.dbNames[name]
-            else f"Database {name} is not selected."
-        )
+        logger.info("Database %s is set as %s.", (name, self.dbNames[name]) if self.dbNames[name]
+                    else "Database %s is not selected.", name)
```

Comment:
Does it run well..?
In my local, it seems to match the formatting incorrectly.

Reply:
For such cases, please use the `if-else` _statement_, not the `expression` for readability (for all the other similar cases as well).

Reply:
I tested it and found log message print well. But I will change the code as @kangz12345 suggested.